debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ApicurioRegistryContainer.java

```diff
@@ -10,15 +10,22 @@
 
 public class ApicurioRegistryContainer extends GenericContainer<ApicurioRegistryContainer> {
 
-    private static final String DEBEZIUM_VERSION = DebeziumContainer.getDebeziumStableVersion();
+    private static final String APICURIO_VERSION = getApicurioVersion();
     private static final Integer APICURIO_PORT = 8080;
+    private static final String TEST_PROPERTY_PREFIX = "debezium.test.";
 
     public ApicurioRegistryContainer() {
-        super("apicurio/apicurio-registry-mem:" + DEBEZIUM_VERSION);
+        super("apicurio/apicurio-registry-mem:" + APICURIO_VERSION);
 
         this.waitStrategy = new LogMessageWaitStrategy()
                 .withRegEx(".*apicurio-registry-app.*started in.*");
 
         addExposedPort(APICURIO_PORT);
     }
+
+    public static String getApicurioVersion() {
+        String APICURIO_VERSION_TEST_PROPERTY = System.getProperty(TEST_PROPERTY_PREFIX + "apicurio.version");
```

Comment:
The uppercasing name should only be used for constants (static final fields), not local vars.